ci-build.sh

```diff
@@ -604,6 +604,34 @@ else
   echo "Testing VERBOSE_ERROR_PAGES works..."
 fi
 
+start_test "Start with http2" "${STD_CMD} \
```

Comment:
could you give this test a move descriptive name? Maybe something like "Test to ensure HTTP/2 is enabled when HTTP2 is set to true